src/baseline/expr.rs

```diff
@@ -1392,13 +1396,49 @@ where
         self.asm.extend_byte(MachineMode::Int64, dest, dest);
     }
 
+    fn emit_intrinsic_unary_call(&mut self, e: &'ast ExprCallType, dest: ExprStore, intr: Intrinsic) {
+        let lhs = e.object.as_ref().unwrap();
+
+        self.emit_intrinsic_unary(lhs, dest, intr, None);
+    }
+
     fn emit_intrinsic_bin_call(&mut self, e: &'ast ExprCallType, dest: ExprStore, intr: Intrinsic) {
         let lhs = e.object.as_ref().unwrap();
         let rhs = &e.args[0];
 
         self.emit_intrinsic_bin(lhs, rhs, dest, intr, None);
     }
 
+    fn emit_intrinsic_unary(
+        &mut self,
+        lhs: &'ast Expr,
+        dest: ExprStore,
+        intr: Intrinsic,
+        op: Option<BinOp>,
```

Comment:
`op` isn't used here, also pretty sure the type is wrong. For unary operators it should be `UnOp`.

Reply:
Will do!